src/win/win-helpers.h

```diff
@@ -12,9 +12,22 @@
 
 #include <windows.h>
 #include <cfgmgr32.h>
+#include "../../common/utilities/os/hresult.h"
 
 #define WAIT_FOR_MUTEX_TIME_OUT  (5000)
 
+#define CHECK_HR_STR(s, x) utilities::hresult::check(s, x, [](std::string descr, bool to_throw)\
```

Comment:
I don't like the way these are set up -- if we didn't already talk about these, talk to me

Reply:
fixed